internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json

```diff
@@ -9977,19 +9971,7 @@
         "Count_Person_15OrMoreYears_ForeignBorn_PlaceOfBirthSouthamerica",
         "Count_Person_15OrMoreYears_ForeignBorn_PlaceOfBirthSouthernEasternEurope",
         "Count_Person_15OrMoreYears_ForeignBorn_PlaceOfBirthWesternAsia",
-        "Count_Person_15OrMoreYears_GoodsSellingUsage_IsInternetUser",
-        "Count_Person_15OrMoreYears_HealthDataResearchUsage_IsInternetUser",
-        "Count_Person_15OrMoreYears_HealthMonitoringServicesUsage_IsInternetUser",
         "Count_Person_15OrMoreYears_HispanicOrLatino",
-        "Count_Person_15OrMoreYears_InternetUsageAtAnyLocation_IsInternetUser",
-        "Count_Person_15OrMoreYears_InternetUsageAtCafeteria_IsInternetUser",
-        "Count_Person_15OrMoreYears_InternetUsageAtHome_IsInternetUser",
-        "Count_Person_15OrMoreYears_InternetUsageAtPublicPlace_IsInternetUser",
-        "Count_Person_15OrMoreYears_InternetUsageAtSchool_IsInternetUser",
-        "Count_Person_15OrMoreYears_InternetUsageAtTraveling_IsInternetUser",
-        "Count_Person_15OrMoreYears_InternetUsageAtWork_IsInternetUser",
-        "Count_Person_15OrMoreYears_JobSearchUsage_IsInternetUser",
-        "Count_Person_15OrMoreYears_LaptopUser",
```

Comment:
@ajaits @hareesh-ms This was from Commerce_NTIA, did we drop these SVs?

Reply:
Yes, this is expected as `armedForcesStatus: dcid:Civilian` was added to convert the SVs to Count_Person_15OrMoreYears_**Civilian**_*